tests/coverage/coverage.cpp

```diff
@@ -811,7 +811,7 @@ TEST(Coverage, generator)
 
     // Coverage for various profile settings.
 
-    analyser->addExternalVariable(libcellml::AnalyserExternalVariable::create(model->component("my_component")->variable("eqnEq")));
+    EXPECT_TRUE(analyser->addExternalVariable(libcellml::AnalyserExternalVariable::create(model->component("my_component")->variable("eqnEq"))));
```

Comment:
given all the tests added to the analyser, is this still needed?

Reply:
in case it got missed, the coverage question here is different to the more general comment...

Reply:
The coverage tests should only be ones that are anomalous, things where we have to cast to `const`, where we are forcing a state simply for coverage’s sake.